components/PageMeta.tsx

```diff
@@ -0,0 +1,57 @@
+import getShareImage from '@jlengstorf/get-share-image';
+import Head from 'next/head';
+import React from 'react';
+
+const site = {
+  title: 'React Native Directory',
+  description: 'An interactive directory to find packages for your React Native apps.',
+};
+
+type PageMetaProps = {
+  title?: string;
+  description?: string;
+  query?: string | string[];
+};
+
+const PageMeta = ({ title, query, description = site.description }: PageMetaProps) => {
+  const pageTitle = `${title ? title + ' • ' : ''}${site.title}`;
+  const parsedQuery = Array.isArray(query) ? query[0] : query;
+  const searchDescription = `Search results for keyword:\n"${parsedQuery}"`;
+
+  const socialImage = getShareImage({
+    title: site.title,
+    tagline: parsedQuery ? searchDescription : description,
+    cloudName: 'react-native-directory',
+    imagePublicID: 'share-bg',
+    titleColor: 'fff',
+    titleFont: 'futura',
+    taglineFont: 'futura',
```

Comment:
Could we make the font match the site?

```
-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif
```

"Helvetica" would do as well!

Reply:
AFAIK Cloudinary supports only Google Fonts out of the box. 

There is an option to used the custom font too, but I'm not sure what exactly licenses the fonts used on the website have and if it's fine to upload them to service like that.

Reply:
@jonsamp What do you think about sticking to Roboto for this PR and we can figure out better solution for that in the future?

Reply:
I have also figure out that I was wrong about fonts support, not all Google Fonts are supported, there are also some other fonts, "Helvetica" also works (but some of the character looked a bit misaligned in the horizontal axis so I have decided to stick to "Roboto"). 

Unfortunately there is also no list of supported fonts and all search results lead to the support pages with vague answers or links to the custom font usage guide.

### Preview
https://res.cloudinary.com/react-native-directory/image/upload/w_1280,h_669,c_fill,q_auto,f_auto/w_760,c_fit,co_rgb:fff,g_south_west,x_480,y_372,l_text:roboto_66:React%20Native%20Directory/w_760,c_fit,co_rgb:aeb1bc,g_north_west,x_480,y_324,l_text:roboto_48:Search%20results%20for%20keyword%3A%0A%22video%22/share-bg